app/src/main/java/com/example/vehicleinfocheck/WebActivity.java

```diff
@@ -79,4 +79,6 @@ private void setSearchButtonListeners() {
             });
         });
     }
-}
\ No newline at end of file
+}
```

Comment:
41 - 'eltVehicleNumber' is not clear, please rename
30 - Variable naming convention is not followed here. Throughout the project camel case (camelCase) was followed except this line where snake case (snake_case) is followed. Also why caps? all of a sudden
59 to 79 - Good piece of code